README.md

```diff
@@ -73,51 +73,76 @@ NOTE, Ideally, we could provision wordpress-trunk into VVV via https://github.co
 
 1. Install phpunit
     ```bash
+    # If not in shell...
     $ vagrant ssh
+    ```
+    In shell...
+    ```bash
```

Comment:
My $0.02 is that this is a bit confusing as it reads "do this" OR "do that". Also, running `vargrant ssh` is done in a shell too..just not the vm shell..   IMO I would just make `vagrant ssh` part of the commands to run, i.e. 

```bash
# SSH into the VM to install PHPUnit
$ vagrant ssh
$ sudo ....
```

Reply:
Good point. Right, maybe change `shell` to `VM` as that is what I meant. Going from step-to-step the confusion might be the person is already in the VM, so that command does not need to be run again unless you exit.